extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -216,7 +215,7 @@ public boolean resume(final String id)
     }
     catch (IOException | InterruptedException e) {
       log.error("Exception [%s] while pausing Task [%s]", e.getMessage(), id);
-      throw Throwables.propagate(e);
+      throw new RuntimeException(e);
```

Comment:
it is odd to log and throw. Such a thing usually leads to double logging. Would it make more sense just to put the log message in the `RuntimeException`?

Reply:
Removed the log and added the message to the RuntimeException